scipy/stats/stats.py

```diff
@@ -3692,6 +3692,103 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+AlexanderGovernResult = namedtuple("AlexanderGovernResult", ("statistic",
+                                                             "pvalue"))
+
+
+def AlexanderGovern(*args):
+    """
+    Performs the Alexander Govern test.
+
+    The Alexander-Govern approximation tests the equality of k independent
+    means in the face of heterogeneity of variance. The test is applied to
+    samples from two or more groups, possibly with differing sizes.
+
+    Parameters
+    ----------
+    sample1, sample2, ... : array_like
+        The sample measurements for each group.  There must be at least
+        two arguments.
+
+    Returns
+    -------
+    statistic : float
+        The computed A statistic of the test.
+    pvalue : float
+        The associated p-value from the chi-squared distribution.
+
```

Comment:
Let's add [See Also](https://numpy.org/doc/stable/docs/howto_document.html) that refers to [f_oneway](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.f_oneway.html) and some Notes that explain the assumptions of the test (like in [f_oneway](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.f_oneway.html). 

Also, it would be great to update the f_oneway docstring to mention this for when there is heterogeneity of variance.